src/Designer/backend/src/Designer/Middleware/UserRequestSynchronization/Abstractions/IHttpContextDataExtractor.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.AspNetCore.Http;
+
+namespace Altinn.Studio.Designer.Middleware.UserRequestSynchronization.Abstractions;
+
+/// <summary>
+/// Service for extracting common data from HTTP context.
+/// Provides reusable methods for resolving organization, developer, and app information.
+/// </summary>
+public interface IHttpContextDataExtractor
```

Comment:
`HttpContext` being in the name and part of the input here, this sounds like something we don't need an interface for perhaps? Unless we have multipe